includes/Modules/Tag_Manager/Settings.php

```diff
@@ -41,6 +43,23 @@ public function register() {
 				'containerId'  => 'containerID',
 			)
 		);
+
+		$this->register_owned_keys( $this->get_owned_keys() );
+	}
+
+	/**
+	 * Returns keys for owned settings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array An array of keys for owned settings.
+	 */
+	public function get_owned_keys() {
+		return array(
+			'accountID',
+			'containerID',
+			'ampContainerID',
```

Comment:
See above, it may be worth including `internalContainerID` and `internalAMPContainerID` here too.

Reply:
Added.